thunder/tests/test_networks.py

```diff
@@ -392,14 +392,14 @@ def test_thunderfx_mistral_nemo_small():
     config = transformers.models.mistral.configuration_mistral.MistralConfig(
         num_hidden_layers=1,
         torch_dtype=torch.bfloat16,
-        max_position_embeddings=1024,
+        max_position_embeddings=64,
         architectures=["MistralForCausalLM"],
-        hidden_size=5120,
+        hidden_size=1024,
         rms_norm_eps=1e-05,
         rope_theta=1000000.0,
         sliding_window=None,
         vocab_size=131072,
```

Comment:
If you'd like to reduce the memory usage even further you may want to scale down vocab_size.